CHANGELOG.md

```diff
@@ -6,11 +6,12 @@ Unreleased (latest)
 
 Changes:
 --------
-- ...
+- Add a config directory named `/jupyter_config_dir` to allow users to mount JupyterLab extension configuration.
 
 Fixes:
 ------
-- ...
+- Use `conda-libmamba-solver` as a default solver to fix resolving issues when installing packages.
```

Comment:
Just out of curiosity, is there more info on the issues you were receiving with the solver?

Reply:
It used to throw an error or would hang during the install, but I've clear my docker cache and try to build again and I dont have the issue anymore, We can revert it to the old way ( .14GB smaller) or keep the change.

Reply:
If the issue is not relevant anymore and the changes are not needed, I would say we can revert those changes.

Reply:
Okay and I found an other package that needs to be to pin to fix an issue. You can check to approve after that.
Also previous tag build was 2.22 GB , building the image today makes it 6.5GB
- This cause eo image to go from 14.2GB to 16.6GB and nlp image to go from 13.7GB to 22.4GB . Do you think the size could cause an issue on the stack or is there enough resources for that?@fmigneault